pkgs/applications/editors/neovim/utils.nix

```diff
@@ -0,0 +1,178 @@
+{ lib
+, vimUtils
+, nodejs
+, neovim-unwrapped
+, bundlerEnv
+, ruby
+, pythonPackages
+, python3Packages
+, writeText
+, wrapNeovim2
+}:
+let
+  # vimWithRC pass it args
+  # need a function that generates wrapperArgs + customRC and then one can create its own package from that.
+  # should let the user write its own file.
+  makeNeovimConfig =
+    {
+    withPython ? false
```

Comment:
I'd be explicit and specify `Python2` at this point.

Reply:
i wonder if we should not drop python2 at this point. Next neovim release doesnt support python2.

Reply:
in fact I should make it even lower level, like just accept a pythonEnv and if null assume python is disabled.

Reply:
Yeah I agree that it might be easiest and best to just drop it.